cli/internal/globby/globby_test.go

```diff
@@ -0,0 +1,520 @@
+package globby
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/spf13/afero"
+)
+
+func setup(files []string) func() {
```

Comment:
I wanted a `setup`/`teardown` function, but there doesn't appear to be a `beforeEach`/`afterEach` type of construct in Go. Is there a more idiomatic way to do this?

Also, this breaks possible test parallelization by setting the state inside of `globby` instead of passing the filesystem to operate on to the function. Open to opinions on this.

Reply:
I didn't know about `testing.Cleanup`, which seems to be the idiomatic way to handle this situation. I previously would've said that each test should `defer cleanup()`.

But also if we do something like my above suggestion re: `init` and `GlobFilesWithFS` I think you won't need the cleanup.

Reply:
Yup, all of this code evaporated. There is a setup call but no teardown since the constructed file system just falls out of scope and gets GCed.